test/extensions/filters/http/ratelimit/ratelimit_test.cc

```diff
@@ -87,6 +87,12 @@ class HttpRateLimitFilterTest : public testing::Test {
   domain: foo
   )EOF";
 
+  const std::string rate_limited_status_config_ = R"EOF(
+  domain: foo
+  rate_limited_status:
+    code: 503
```

Comment:
It would be great to test the case of status code < 400

Reply:
ok,i think it's good to add another test for this case?I will do it later,thanks~

Reply:
@soulxu I have add the test for case of status code < 400,PTAL,thanks~